src/main/groovy/com/github/prokod/gradle/crossbuild/ResolutionStrategyConfigurer.groovy

```diff
@@ -69,46 +72,49 @@ class ResolutionStrategyConfigurer {
             def dependencySets = insightsView.getDependencySets(DependencySetType.ALL)
 
             def allDependencies = dependencySets.flatMapped()
-            project.logger.info(LoggerUtils.logTemplate(project,
-                    lifecycle:'afterEvaluate',
-                    configuration:crossBuildConfigurationName,
-                    parentConfiguration:names.main,
-                    msg:"Inherited dependendencies to consider while resolving ${crossBuildConfigurationName} " +
-                            'configuration dependencies: [' +
-                            "${allDependencies.collect { "${it.group}:${it.name}" }.join(', ')}]"
-            ))
 
+            //avoid big string creation
+            if (project.logger.infoEnabled) {
```

Comment:
What is the performance hit when info is enabled?

Reply:
I don't know exact but making a string with all dependencies probably gives some extra GC overhead.